### PR TITLE
Extract metadata from pool helpers

### PR DESCRIPTION
**Pull request by agualis**
# Description

This is a preparatory PR to boost app loading time. 

We want to avoid that router handlers like `meta.service.ts `depend on SDK so that we extract metadata logic to make it independent of global Pool logic. 

At the same time, we move metadata logic closer to the config (metadata is defined in config files) so that there's more cohesion. 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency changes
- [x] Code refactor / cleanup
- [ ] Documentation or wording changes
- [ ] Other

## How should this be tested?

Please provide instructions so we can test. Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have requested at least 1 review (If the PR is significant enough, use best judgement here)
- [ ] I have commented my code where relevant, particularly in hard-to-understand areas
- [ ] If package-lock.json has changes, it was intentional.
- [ ] The base of this PR is `master` if hotfix, `develop` if not

### CODE REVIEW

File: src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue [hunk 1]
```diff
@@ -252,12 +251,10 @@ function getPickedTokens(tokens: PoolToken[]) {
...
         </div>
       </template>
-      <template
-        #poolCompositionCell="{ pool, address, addedTimestamp, network }"
-      >
+      <template #poolCompositionCell="{ pool, address, addedTimestamp }">
         <div v-if="!isLoading" class="flex items-center py-4 px-6">
-          <div v-if="poolMetadata(pool.id, network)" class="text-left">
-            {{ poolMetadata(pool.id, network)?.name }}
+          <div v-if="poolMetadata(pool.id)" class="text-left">
```
Comment (agualis) [inline src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue:256]: @alter-eggo As you can see I have removed second parameter network from `poolMetadata` function. This is the only place where we were using it and I don't think we need it but I'm not very familiar with this Gauge component and I'm not sure how to test it. Could you please double check?

Comment (alter-eggo) [reply src/components/contextual/pages/vebal/LMVoting/GaugesTable.vue:256]: yeah, after @timjrobinson refactoring there is no need in network param

File: src/components/pool/PoolPageHeader.vue [hunk 2]
```diff
@@ -120,8 +121,7 @@ const poolTypeLabel = computed(() => {
   return key ? t(key) : t('unknownPoolType');
 });
 
-const poolMetadata = computed(() => POOLS.Metadata[props.pool?.id]);
-const hasMetadata = computed((): boolean => !!poolMetadata.value);
+const hasMetadata = !!poolMetadata;
```
[the remark this answers is not in the document]
Comment (agualis) [reply src/components/pool/PoolPageHeader.vue]: My fault. I fixed it.

File: src/services/meta/meta.service.ts [hunk 3]
```diff
@@ -1,13 +1,20 @@
...
 import { RouteLocationNormalized } from 'vue-router';
 import { configService } from '../config/config.service';
 import { ROUTE_META_DATA } from './meta.constants';
-import { poolMetadata } from '@/composables/usePoolHelpers';
 
 interface IMetaService {
   setMeta(route: RouteLocationNormalized | string): void;
 }
 
+// We use this interface to avoid importing Pool type from SDK while it is not tree-shakable
```
Comment (alter-eggo) [inline src/services/meta/meta.service.ts:10]: 👍

**Review by alter-eggo — APPROVED** [review]

**Pull request merged**